circle.yml

```diff
@@ -6,15 +6,23 @@ general:
       - production
 machine:
   post:
-    - npm install jscs@1.13.1 -g
-    - npm install mocha@2.4.5 -g
+    - npm install jscs@1.13.1 mocha@2.4.5 -g
+    - cd api && npm run test:e2e website install
```

Comment:
Alright then, you got me. That means we should rename `test/frontend` to `test/e2e` on the website repo.